docs/R_USER_DOCUMENTATION.md

```diff
@@ -37,22 +37,20 @@ a hosted repository.
 You can set up an R proxy repository to access a remote repository location, for example the official R
 registry at [https://cran.r-project.org/](https://cran.r-project.org/) that is configured as the default in R.
 
-To proxy a R repository, you simply create a new 'r (proxy)' as documented in [Repository Management](https://help.sonatype.com/display/NXRM3/Configuration#Configuration-RepositoryManagement) in
-details. Minimal configuration steps are:
+To proxy a R repository, you simply create a new 'r (proxy)' as shown in the documented example in 
```

Comment:
Pedantic: I hate when documentation implies something should be simple when it is often not.  I'd rephrase it without "simply".

Reply:
Will boyscout, no-one can stop me!=D